src/Command/LazyLoadingCommand.php

```diff
@@ -50,12 +50,22 @@ public function __construct(string $name, string $commandClass, ContainerInterfa
         $this->setHelp($command->getHelp());
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output) : int
+    /**
+     * @return int
+     */
+    public function run(InputInterface $input, OutputInterface $output)
     {
-        return $this->getCommand()->execute($input, $output);
+        return $this->getCommand()->run($input, $output);
     }
 
-    private function getCommand() : parent
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        // intentionally empty; method will never be called, as `run()` has been
+        // overridden.
+        return 0;
```

Comment:
if this method is never called, I would suggest to throw exception here, so we know that something went wrong.